packages/elements/src/datetime-field/index.ts

```diff
@@ -0,0 +1,708 @@
+              break;
+            case 'weekday':
+              newValue = iterateUnit(this.value, 'day', amount);
+              break;
+            case 'dayPeriod':
+              newValue = iterateUnit(this.value, 'hour', amount * 12);
+              break;
+            case 'fractionalSecond':
+              newValue = iterateUnit(this.value, 'millisecond', amount * 100);
+              break;
+            // no default
+          }
+          this.setValueAndNotify(newValue);
+          this.syncInputValue();
+          this.selectPart(selectedPartIndex, this.formatToParts(newValue));
+          this.resetError();
+          event.preventDefault();
+        }
+        break;
+      case 'ArrowRight':
+      case 'ArrowLeft':
+        const nextPartIdx = getNextSelectedPartIndex(selection, parts, this.inputValue, key === 'ArrowLeft' ? Direction.Down : Direction.Up);
+        this.selectPartFrame.schedule(() => {
+          this.selectPart(nextPartIdx, parts);
+        });
+        break;
+      // no default
+    }
+  }
+
+  /**
+   * Check if value is changed and fire event
+   * @returns {void}
+   */
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  protected override onPossibleValueChange (event: InputEvent): void {
+    // Nobody likes to see a red border
+    this.resetError();
+    this.selectPartFrame.cancel(); // ensure no pending selection
+    this.partLabel = '';
+
+    const inputValue = this.inputElement?.value || '';
+    const parsedValue = this.toValue(inputValue);
+    this.setValueAndNotify(parsedValue);
+  }
+
+  /**
+   * Handles key down input event
+   * @param event Key down event object
+   * @returns {void}
+   */
+  protected onInputKeyDown (event: KeyboardEvent): void {
+    if (this.readonly || this.disabled || event.defaultPrevented) {
+      return;
+    }
+
+    // Ignore special keys
+    if (event.shiftKey || event.ctrlKey || event.altKey || event.metaKey) {
+      return;
+    }
+
+    switch (event.key) {
+      case 'Up': // IE11
+      case 'ArrowUp':
+        this.onNavigation('ArrowUp', event);
+        break;
+      case 'Down':
+      case 'ArrowDown':
+        this.onNavigation('ArrowDown', event);
+        break;
+      case 'Left':
+      case 'ArrowLeft':
+        this.onNavigation('ArrowLeft', event);
+        break;
+      case 'Right':
+      case 'ArrowRight':
+        this.onNavigation('ArrowRight', event);
+        break;
+      case 'Esc':
+      case 'Escape':
+        this.onNavigation('Escape', event);
+        break;
+      default:
+        return;
+    }
+  }
+
+  /**
+   * Decorate `<input>` element with common properties.
+   * Override placeholder
+   * @returns template map
+   */
+  protected override get decorateInputMap (): TemplateMap {
+    return {
+      ...super.decorateInputMap,
+      placeholder: this.focused && !this.inputValue ? this.toInputValue(this.startDate) : this.placeholder || undefined,
```

Comment:
I not sure of replacing placeholder set by the user with start date value is a good idea. Placeholder is something that would provide some kind of a hint or information about the field which we should not override.

Reply:
And from accessibility point of view, placeholders should not be used to describe the field, instead labels should be used

Reply:
I still think placeholder set by user should be read when element is in focus. Perhaps we should discuss this in the meeting.

Reply:
OK, we can discuss

Reply:
as discussed, placeholder is removed